CONTRIBUTING.md

```diff
@@ -114,63 +200,136 @@ Tags disponíveis:
+2. Verifique se a ideia já não está presente em nossas [issues do GitHub](https://github.com/levxyca/diciotech/issues)
 
 Concluindo esses dois passos, você pode [criar uma nova issue](https://github.com/levxyca/diciotech/issues/new) descrevendo as melhorias e usando o label **feature**.
 
-## Discutindo as issues
+## ⚒️ Instalando o ambiente de desenvolvimento para contribuir
 
-Antes de partirmos para o código em si é muito importante discutirmos com a comunidade como cada issue será abordada.
+Para adicionar novos termos técnicos não é necessário instalar o ambiente de desenvolvimento. Todo o processo pode ser feito pela própria interface web do GitHub.
 
-Issues que estão em processo de discussão devem receber o label **discussion**, indicando que aquela issue precisa de feedbacks da comunidade.
+Já para contribuições de layout ou qualquer outra, você precisa instalar o ambiente de desenvolvimento. O `diciotech` é construído com Ruby e Jekyll, então você precisa ter o Ruby instalado em sua máquina. Aqui estão as instruções para instalar o ambiente de desenvolvimento:
 
-## Fazendo pull requests
+### Usando Development Containers (recomendado)
 
-Antes de abrir o seu PR, faça um fork do projeto e trabalhe em cima de um branch diferente da `main`, implementando suas soluções. Para saber mais sobre pull requests e como eles funcionam, veja [este link](https://help.github.com/articles/about-pull-requests/).
+O `diciotech` suporta [Development Containers](https://containers.dev/supporting). Para configurá-lo, siga as etapas abaixo:
 
-Antes de abrir seu PR (pull request):
+1. Certifique-se de ter o [Docker](https://www.docker.com/products/docker-desktop) instalado e em execução no seu sistema.
+2. Abra o repositório do projeto no **Visual Studio Code (VSCode)**.
+3. O VSCode solicitará que você instale a extensão necessária. Clique em "Instalar" e aguarde a configuração automática (este processo pode demorar na primeira vez).
+4. Após a instalação, clique na opção **"Reopen in Container"** quando solicitado. Novamente, aguarde enquanto o ambiente é configurado (pode levar mais tempo na primeira execução).
+5. Quando a conexão com o DevContainer for concluída, o Diciotech estará pronto para uso.
 
-- Leia com atenção o [README](./README.md) do projeto;
-- Se atente para que tenha um issue aberta relacionada ao seu PR;
-- Caso não tenha, crie uma seguindo o guia de contribuição.
+#### Visualizando o site em execução
+
+1. Abra seu navegador e acesse: `http://localhost:4000`.
+2. Você verá uma cópia do [site oficial](https://diciotech.netlify.app/).
+
+Agora você pode personalizar o site conforme necessário. Após realizar as alterações, lembre-se de **commitar** suas mudanças finais.
+
+### Instalando manualmente
+
+A instalação manual é um pouco mais trabalhosa, mas é útil se você deseja ter mais controle sobre o ambiente de desenvolvimento. Recomendado somente para usuários avançados.
+
+#### Instalando Ruby e rbenv
```

Comment:
Aqui eu não ensinaria a instalar Ruby ou rbenv, uma vez que depende do S.O do dev. Talvez só citaria que é necessário instalar o Ruby na versão X.

Reply:
Eu pensei nisso, mas achei útil dar o caminho das pedras pra quem prefere enfrentar esse caminho. Ou que por algum motivo não pode instalar ferramentas a nível global na máquina que vai testar por falta de permissões.

Reply:
Então, mas se for um usuário avançado, ele não precisaria dessa instrução. Eu acho que vira mais um ponto para manter atualizado no guia. Mas ai é minha opinião apenas =)

Reply:
Usuário avançado = disposto a instalar as coisas no braço e acostumado a fazer isso. Não exatamente avançado no uso do jekyll.

Fora que eu acho um saco quando eu vou num repositório tentar instalar o ambiente e ele não explica direito nem direciona pra algum lugar que tenha um explicação direta e que funcione.